test/test_dataset_attributes.py

```diff
@@ -0,0 +1,31 @@
+"""
```

Comment:
I took the liberty of beginning a test file which can be extended in the future.  It could also be included in some kind of automatic testing pipeline if you ever choose to set one up.